libr/io/submap.c

```diff
@@ -0,0 +1,289 @@
+#include <r_io.h>
+#include <r_types.h>
+#include <r_util.h>
+
+ut64 _map_from(RIOMap *map) {
+	return map->itv.addr;
+}
+
+ut64 _map_to(RIOMap *map) {
+	return map->itv.addr + map->itv.size - 1;
+}
+
+//in - incoming
+static bool _submap_find_cb(void *vin, void *vincoming, void *user, int *cmp_res) {
```

Comment:
This function can only return `true`, cmp_res being an output parameter is unnecessary